### ISSUE TITLE
Copy graph styling when adding new graph

### ISSUE THREAD

**Issue by nickerso** [opencor/opencor#1485]
Now that we have a whole slew of awesome graph configuration options that can take a while to get set up, it might be nice if you could get one graph panel set up how you like it and then when adding new panels the settings from the currently selected one get copied.

**Comment by agarny** [comment]
Indeed, I agree that it would make sense to have such a feature. I guess that, in the meantime, we will have to do with issue #1366 (which should be completed sometime this week, and will trigger a new snapshot).
